android/src/main/java/com/sailthru/mobile/rnsdk/RNSailthruMobileModule.java

```diff
@@ -88,13 +88,7 @@ public boolean shouldPresentInAppNotification(@NonNull Message message) {
                     .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
                     .emit("inappnotification", writableMap);
 
-        } catch (NoSuchMethodException e) {
-            e.printStackTrace();
-        } catch (IllegalAccessException e) {
-            e.printStackTrace();
-        } catch (JSONException e) {
-            e.printStackTrace();
-        } catch (InvocationTargetException e) {
+        } catch (NoSuchMethodException | IllegalAccessException | JSONException | InvocationTargetException e) {
             e.printStackTrace();
```

Comment:
I know this isn't part of the RP but ... is this good practice? I thought that android swallows stderr by default. are we doing anything to assist API users if this happens?

Reply:
It does look like it would be better if we updated it to log properly for Android, I'll double check this doesn't surface properly in RN logs first though.